pvm.py

```diff
@@ -36,29 +37,90 @@ def _init_logger():
 VIDEO_PATH = "jellyfish720p.mp4"
 media = ""
 IS_FILE_SET = False
+NEXT_TIME = None
+
+'''
+UDP command example:
+/PVM 12:13:14
+/PVM <action_name> <action_value>
+
+The normal workflow is below,
+1. Receive the first command: /PVM 12:13:14
+	- parse the time
+	- plus 3s
+	- save the time into NEXT_TIME
+2. Receive the second command: /PVM file jellyfish.mp4
+	- Wait until the NEXT_TIME
+	- Execute the command
+	- Set NEXT_TIME = None
+'''
 
 def parse_commands(*args):
 	global media
 	global VIDEO_PATH
 	global IS_FILE_SET
+	global NEXT_TIME
+	# Parse the time send in UDP, 
+	# From /PVM 12 13 14 to 12:13:14
+	if len(args) == 4:
+		seconds = int(args[3]) + 3
+		minutes = int(args[2])
+		hours = int(args[1])
```

Comment:
yup, I can definitely see this being very problematic here, if we have some other command that also has 3 values then it will be parsed here and the program may not like that surprize.
```suggestion
	if args[1]=="timestamp" && len(args) == 5:
		seconds = int(args[4]) + 3
		minutes = int(args[3])
		hours = int(args[2])
```

Reply:
following the new commit https://github.com/omarcostahamido/PVM/pull/38/commits/33404ff646efb2093db36b51a4a6df5158cb5c23 this would have to be different now

Reply:
Got it. Will change and test ASAP.